.stylelintrc

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": ["stylelint-config-recommended", "stylelint-config-idiomatic-order", "stylelint-config-standard-scss"],
+  "extends": ["stylelint-config-recommended", "stylelint-config-standard-scss"],
```

Comment:
Please revert this file to its original form and address the potential style errors:
- https://github.com/AlaskaAirlines/WC-Generator/blob/master/template/.stylelintrc